include/random_walks/shake_and_bake_walk.hpp

```diff
@@ -0,0 +1,169 @@
+// VolEsti (volume computation and sampling library)
```

Comment:
For each new file you add in the codebase, I think correct mark should be:

```
// VolEsti (volume computation and sampling library)

// Copyright (c) 2012-2025 Vissarion Fisikopoulos
// Copyright (c) 2018-2025 Apostolos Chalkis
// Copyright (c) 2025-2025 Iva Janković

// Contributed and/or modified by Iva Janković, as part of Google Summer of
// Code 2025 program.

// Licensed under GNU LGPL.3, see LICENCE file
```

Sorry if I misspelled your name :)

Reply:
@vfisikop can you confirm this?